### PR TITLE
Fixes after validation of TypeInstances

### PR DESCRIPTION
**Pull request by mkuziemko**
## Description
Changes proposed in this pull request:
- change port in implementation which uses Postgres to be number (not string),
- remote tablespace in Postgres - not used by TypeInstance,
- in capact/validation change key to be string, not boolean as mainly that how we use it in integration tests. 

## Related issue(s)
- https://github.com/capactio/capact/issues/544

### CODE REVIEW

File: manifests/type/capactio/capact/validation/upload.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ spec:
         "title": "The schema for upload Type",
         "examples": [
```
Comment (pkosiec) [inline manifests/type/capactio/capact/validation/upload.yaml:22]: The Implementation `passing-a` uploads TypeInstance with `key: testValue`. Should the `passing-b` Implementation be changed as well? Cause I see there:

```yaml
              args: ["echo 'Implementation B' && cp /input /upload && echo 'key: true' > /expected && diff /expected /injected && sleep 1"]

```

Comment (mkuziemko) [reply manifests/type/capactio/capact/validation/upload.yaml:22]: Not really. In our e2e tests, we examine if the expected is the same as injected for the value `key: true` so it is ok.

Comment (pkosiec) [reply manifests/type/capactio/capact/validation/upload.yaml:22]: Oh, right 👍  I read the code in wrong way. Sounds good 👍

**Review by pkosiec — APPROVED** [review]
LGTM

**Pull request merged**